src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public int getRandomBall() {
```

Comment:
method getRandomBall() should return instance of Ball class with random number and random color